appveyor.yml

```diff
@@ -1,3 +1,4 @@
+os: unstable  
```

Comment:
Can you add a comment here that this causes the use of WMF5?
[appveyor yml docs](http://www.appveyor.com/docs/appveyor-yml)